### ISSUE TITLE
Where to find maps?

### ISSUE THREAD

**Issue by SILVERWAVE20** [Zet0rz/nzombies#593]
I just installed the game onto my new computer and I just wanted to know, where would I find the maps folder so I can put the local maps that I made onto my new computer?

**Comment by minin43** [comment]
Your question doesn't make a lot of sense. Are you asking a general question about Garry's Mod? As in: where is Garry's Mod's maps folder?

**Comment by SILVERWAVE20** [comment]
no, in the files theres a map folder for nzombies, and I want to know where the folder is located.

**Comment by sh4d0w59** [comment]
Nzombies has a map folder that houses the navigation files for the official maps you get off of the workshop but that's it.  If you're wondering where you store local maps (aka the raw bsp files you made yourself), you put them in "C:\Program Files (x86)\Steam\steamapps\common\GarrysMod\garrysmod\maps"
